front/app/components/CampaignConsentForm/feedback.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react';
+import Error from 'components/UI/Error';
+import {
+  Text,
+  Title,
+  Box,
+  colors,
+  Icon,
+} from '@citizenlab/cl2-component-library';
+import { scrollToElement } from 'utils/scroll';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+type FeedbackProps = {
+  successMessage: string;
+  showFeedback: false | 'success' | 'error';
+  errorMessage: string;
+  closeFeedback: () => void;
+};
+
+const Feedback = ({
```

Comment:
Did you try using this component `front/app/components/HookForm/Feedback/index.tsx` instead of a new one?

Reply:
yes, but is too tightly coupled with react hook form, and was taking me a lot of time trying to make this work as a react form. so i cloned the component and made it simpler.

Reply:
I see. Then maybe at least some parts can be reused? E.g., this piece looks like a good common component

```ts
const successFeedback = ({ successMessage, closeSuccessMessage }) => {
  return (
    <Box
      bgColor={colors.successLight}
      borderRadius="3px"
      px="12px"
      py="4px"
      mb="12px"
      role="alert"
      display="flex"
      justifyContent="space-between"
      alignItems="center"
      data-testid="feedbackSuccessMessage"
    >
      <Box display="flex" gap="16px" alignItems="center">
        <Icon
          name="check-circle"
          fill={colors.success}
          width="24px"
          height="24px"
        />
        <Title color="success" variant="h4" as="h3">
          {successMessage || formatMessage(messages.successMessage)}
        </Title>
      </Box>
      <CloseIconButton onClick={closeSuccessMessage} />
    </Box>
  );
};
```

Without it, it looks like the new component is something new and unique and not related to anything else. In the future, it will diverge from the original one, and bug fixes and UI changes will be applied in one but not another.

Not a blocker though.